Description: Fix spelling error in pki_* code
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
--- a/src/pki_crypto.c
+++ b/src/pki_crypto.c
@@ -653,7 +653,7 @@
             return NULL;
         case SSH_KEYTYPE_UNKNOWN:
             BIO_free(mem);
-            ssh_pki_log("Unkown or invalid private key type %d", key->type);
+            ssh_pki_log("Unknown or invalid private key type %d", key->type);
             return NULL;
     }
 
@@ -779,7 +779,7 @@
             /* Cannot open ed25519 keys with libcrypto */
         case SSH_KEYTYPE_UNKNOWN:
             BIO_free(mem);
-            ssh_pki_log("Unkown or invalid private key type %d", type);
+            ssh_pki_log("Unknown or invalid private key type %d", type);
             return NULL;
     }
 
--- a/src/pki_gcrypt.c
+++ b/src/pki_gcrypt.c
@@ -671,7 +671,7 @@
         case SSH_KEYTYPE_ECDSA:
         case SSH_KEYTYPE_UNKNOWN:
         default:
-            ssh_pki_log("Unkown or invalid private key type %d", type);
+            ssh_pki_log("Unknown or invalid private key type %d", type);
             return NULL;
     }
 
